src/components/screen/ReleasesScreen.jsx

```diff
@@ -16,22 +16,18 @@ import {
   MenuItem
 } from "@material-ui/core";
 import styled from "styled-components";
-import {
-  LocalOffer,
-  CalendarToday,
-  Sort
-} from "@material-ui/icons";
+import { LocalOffer, CalendarToday, Sort } from "@material-ui/icons";
 import Ingress from "../shared/Ingress";
 import { fetchArticles } from "../../actions/articleActions";
 import { useDispatch, useSelector } from "react-redux";
 import { useLocation } from "react-router";
 import articleParameters, { sortKeys } from "../../articleParameters";
 
 const ReleasesScreen = props => {
-  const [anchorEl, setAnchorEl] = React.useState(null)
+  const [anchorEl, setAnchorEl] = React.useState(null);
```

Comment:
Brukes *anchorEl* og *setAnchoEl* lengre? Eg brukt det for lenge sia når eg sku lag den "Velg Produkt".

Reply:
Velg produkt e der fortsatt